tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/document/documents.controller.js

```diff
@@ -937,7 +967,7 @@
                         //vm.nodata = false;
                         vm.lazyloaderFilter = true;
                         if (bool) {
-                            vm.gridOptions.data = response;
+                            vm.showDocumentAsPinOrUnpin(searchRequest, response)

```

Comment:
Added one reusable method which will be called at several places for pin and unpin functionality